python/feathub/processors/local/local_processor.py

```diff
@@ -178,26 +211,41 @@ def materialize_features(
                 timestamp_field=features.timestamp_field,
                 timestamp_format=features.timestamp_format,
             )
-        elif isinstance(sink, FileSystemSink):
+        elif isinstance(sink, FileSystemSink) and _is_local_file_or_dir(sink.path):
             insert_into_file_sink(features_df, sink)
             return LocalJob()
 
-        raise RuntimeError(f"Unsupported sink: {sink}.")
+        self._init_spark_session_local_mode()
```

Comment:
Would it be better to make the code style consistent with `_get_table(...)` by e.g. using `elseif`?